tests/foreman/cli/test_errata.py

```diff
@@ -383,12 +829,33 @@ def test_positive_list_filter_by_product_id(self):
         1. erratum list --product-id=<productid>
 
         @Assert: Errata is filtered by product id.
-
-        @caseautomation: notautomated
-
         """
-
-    @stubbed()
+        product_erratum_list = Erratum.list({
+            'product-id': self.org_product['id'],
+            'per-page': 1000
+        })
+        product_errata_ids = {
+            errata['errata-id']
+            for errata in product_erratum_list
+            }
+        product2_erratum_list = Erratum.list({
+            'product-id': self.org2_product['id'],
+            'per-page': 1000
+        })
+        product2_errata_ids = {
+            errata['errata-id']
+            for errata in product2_erratum_list
+        }
+        self.assertEquals(len(product_errata_ids), 4)
+        self.assertEquals(len(product2_errata_ids), 4)
+        self.assertIn(FAKE_2_ERRATA_ID, product_errata_ids)
+        self.assertIn(FAKE_1_ERRATA_ID, product2_errata_ids)
+        self.assertSetEqual(
```

Comment:
`assertEqual()` will analyze arguments and in case sets were passed - will call `assertSetEqual`. According to unittest docs you shouldn't use `assertSetEqual` directly and prefer using `assertEqual()` instead.